<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MeghaSinha" name="Default test(failed)" tests="6" failures="1" timestamp="5 Mar 2019 08:44:31 GMT" time="0.063" errors="0">
  <testcase name="run" time="0.016" classname="FailedTestCase.Demo1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="FailedTestCase.Demo1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.001" classname="FailedTestCase.Demo1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.002" classname="FailedTestCase.Demo1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.001" classname="FailedTestCase.Demo1">
    <skipped/>
  </testcase> <!-- run -->
  <testcase name="run" time="0.001" classname="FailedTestCase.Demo1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at FailedTestCase.Demo1.run(Demo1.java:9)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- run -->
</testsuite> <!-- Default test(failed) -->
